### PR TITLE
add Dynamic-Insert.md

### PR DESCRIPTION
**Pull request by linrongbin16**
Hi @rbock , I'm trying to add document on `dynamic_insert_into` API (also I'm developing my app using this API).

Would you please help review this PR?

### CODE REVIEW

File: docs/Dynamic-Insert.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# A Basic Example
+So let us construct insert statement with a dynamic part
+```C++
+auto s = dynamic_insert_into(db, foo).dynamic_set(
+    foo.name = name,
+);
+s.insert_list.add(foo.id = runtimeStatement.id);
+s.insert_list.add(foo.hasFun = runtimeStatement.hasFun);
+```
+Admittedly, a rather lame example (please suggest nicer ones), but anyway, this is what's happening:
```
Comment (rbock) [inline docs/Dynamic-Insert.md:15]: Maybe replace "lame" with "simplistic"?

**Comment by rbock** [comment]
Thanks for the PR! It would be great if you could add a line that actually executes the insert statement.

File: docs/Dynamic-Insert.md (hunk 1 above)
Comment (linrongbin16) [reply docs/Dynamic-Insert.md:15]: sure

**Comment by linrongbin16** [comment]
> Thanks for the PR! It would be great if you could add a line that actually executes the insert statement.

Do you mean add `db(s)` at last in the sample code ?
Just like:
```C++
auto s = dynamic_insert_into(db, foo).dynamic_set(
    foo.name = name,
);
s.insert_list.add(foo.id = runtimeStatement.id);
s.insert_list.add(foo.hasFun = runtimeStatement.hasFun);
const int64_t result = db(s);    // <<< add this line ?
```

**Comment by rbock** [comment]
Yes, a line like that and a comment what it is doing would be nice.

**Comment by linrongbin16** [comment]
> Yes, a line like that and a comment what it is doing would be nice.

I have edit my PR, would you please check again?

**Comment by rbock** [comment]
Thanks for the quick adjustments!

**Pull request merged**